cost/google/cud_expiration/README.md

```diff
@@ -1,32 +1,38 @@
-## Google Expiring Committed Use Discount (CUD)
+# Google Expiring Committed Use Discount (CUD)
+
+## What it does
 
-### What it does
 This policy identifies all active CUDs that exist in a given GCP project that will be expiring in a set number of days.
 
-### Functional Details
+## Functional Details
 
 - Uses the GCP API to get a list of all CUDs and sends a notification.
-- Create a service account (if none exist) with `owner` role under Google-cloud platform (IAM & admin -> service accounts). Generate key, a JSON file will get downloaded in which you can find 'client email' and 'private key' which has to be added as credentials in RightScale cloud management Design -> Credentials with name 'GC_SA_CLIENT_EMAIL' and 'GC_SA_PRIVATE_KEY' respectively.  
+- Create a service account (if none exist) with `owner` role under Google-cloud platform (IAM & admin -> service accounts). Generate key, a JSON file will get downloaded in which you can find 'client email' and 'private key' which has to be added as credentials in RightScale cloud management Design -> Credentials with name 'GC_SA_CLIENT_EMAIL' and 'GC_SA_PRIVATE_KEY' respectively.
```

Comment:
This part is no longer needed.

Reply:
I have addressed the comment. Please review the changes.